index.bs

```diff
@@ -145,7 +145,8 @@ Unless specified otherwise, these steps are:
 
    Note: [[#native-file-system-permissions]] overrides these steps for entries returned by
    the [=native file system handle factories=]. Additionally [[#sandboxed-filesystem-concepts]] overrides
-   these steps for entries returned by {{getOriginPrivateDirectory()}}.
+   these steps for entries returned by {{getOriginPrivateDirectory()}}, and the gesture of dragging
+   and dropping overrides these steps for entries dropped onto the web.
```

Comment:
(after making drag&drop be a subsection of accessing native file system), I'd rephrase this as 

[[#nfs-permissions]] overrides these steps for entries returned by factories, as well as entries exposed via drag and drop

With a link to the drag&drop section,